recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - wheel
     # These are also needed at build time.
     - bleach
-    - bokeh <2.5.0
+    - bokeh 2.4.3
     - nodejs 
     - pyct
     - packaging
```

Comment:
Could add:
```
run_constrained:
  - holoviews >1.14.1
```

Reply:
Thanks for pointing this out, @cbouss. Did some reading and I agree with the suggestion.